_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Integrating alerts with Dashboard 
+parent: Alerting
+nav_order: 50
+---
+
+# Integrating alerts with Dashboard
+Introduced 2.8
+{: .label .label-purple }
+
+Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly. Use the **Dashboard** app to:
+
+- Set up, add, and adjust rules and conditions that trigger alerts and notifications.
+- Create graphs that show trends and patterns and build intuitive dashboards to stay on top of important metrics and data points in real time.
+- Monitor your alerts in one place with at-a-glance views.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/alerting-viz.png" alt="Example alerting visualization" width="800" height="800">
+
+## Getting started 
+
+Before getting started, you must have:
+
+- Installed OpenSearch and OpenSearch Dashboards version 2.8 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
+- Installed Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+
+## Configuring admin settings
+
+You can only access, create, or manage alerts for resources for which you have permissions. Access to alerting dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards privileges, and you can manage the settings in **Stack Management**. Access is enabled by default and appears as a feature in the **Stack Management** > **Advanced Settings** > **Visualizations** window. If the setting is disabled, it does not appear in this window. The setting is disabled at the cluster level through the `opensearch-dashboards.yml` file.
```

Comment:
The setting "can be" disabled at the cluster level through the `opensearch-dashboards.yml` file